### DISCUSSION TITLE
Tasmota integration add units to an entity ?

### DISCUSSION THREAD

**Discussion by Yachtmed** [home-assistant/frontend#21252] [Configuration] [answered]
Tasmota normally running on a memory limited device does not add units to mqtt, except for deg C or F.
I have the output of an ADC channel in volts.
Because there is no units, any graph from this is horizontal multi coloured bar chart instead of a line graph.

Any way of adding required units to an entity within an integration ?

**Comment by karwosts** [comment]
Don't know about tasmota specifically, but you can add a unit_of_measurement to an entity with `customize:` as a last resort. 

https://www.home-assistant.io/integrations/homeassistant/#manual-customization

**Reply by Yachtmed** [reply, marked as answer]
It seemed to be a glitch in 2024.4.6, update to .7 seems to have fixed it
